File: samples/python/sample.py
#!/usr/bin/env python2

import sys


# UT{ calculate
def calculate(value):
    return value * value
# UT}


# UT{ entry_point
def entry_point():
    if len(sys.argv) != 2:
        print "Wrong arguments. Should be one number"
        return 1
    value = int(sys.argv[1])
    print "Input value =", value
    print "Result =", calculate(value)
    return 0
# UT}


if __name__ == "__main__":
    sys.exit(entry_point())

File: samples/go/sample.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

// UT{ calculate
func calculate(value int) int {
	return value * value
}

// UT}

// UT{ entry_point
func entry_point() int {
	if len(os.Args) != 2 {
		fmt.Println("Wrong arguments. Should be one number")
		return 1
	}
	value, _ := strconv.Atoi(os.Args[1])
	fmt.Printf("Input value = %d\n", value)
	fmt.Printf("Result = %d\n", calculate(value))
	return 0
}

// UT}

func main() {
	os.Exit(entry_point())
}
